src/ui/MobileSearch/Header.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+  import Svg from '@/ui/Svg/svelte'
+  import InputWithIcon from '@/ui/InputWithIcon.svelte'
+
+  export let show
+  export let searchTerm
+
+  function onInput({ currentTarget }) {
+    searchTerm = currentTarget.value
+  }
+</script>
+
+<div class="wrapper row v-center mrg-l mrg--b">
+  <button class="arrow btn row hv-center mrg-s mrg--r" on:click={() => (show = false)}>
+    <Svg id="arrow-left-big" w="11" h="18" />
+  </button>
+  <div class="relative fluid input-wrapper">
+    <InputWithIcon
+      value={searchTerm}
+      on:input={onInput}
+      placeholder="Search for assets, trends..."
+      big
+      class="$style.input body-2"
+    >
+      <div class="search row hv-center" slot="left">
+        <Svg id="search" w="12" />
+      </div>
```

Comment:
Better to use custom render if the default `icon` of `InputWithIcon` is not used

Reply:
@Woafflation not resolved